ush/python/pygfs/task/snow_analysis.py

```diff
@@ -306,250 +335,85 @@ def finalize(self) -> None:
                 anllist.append([src, dest])
         FileHandler({'copy': anllist}).sync()
 
-        if config.DOIAU:
+        if self.task_config.DOIAU:
             logger.info("Copying increments to beginning of window")
-            template_in = f'snowinc.{to_fv3time(config.current_cycle)}.sfc_data.tile{{tilenum}}.nc'
-            template_out = f'snowinc.{to_fv3time(config.SNOW_WINDOW_BEGIN)}.sfc_data.tile{{tilenum}}.nc'
+            template_in = f'snowinc.{to_fv3time(self.task_config.current_cycle)}.sfc_data.tile{{tilenum}}.nc'
+            template_out = f'snowinc.{to_fv3time(self.task_config.SNOW_WINDOW_BEGIN)}.sfc_data.tile{{tilenum}}.nc'
             inclist = []
-            for itile in range(1, config.ntiles + 1):
+            for itile in range(1, self.task_config.ntiles + 1):
                 filename_in = template_in.format(tilenum=itile)
                 filename_out = template_out.format(tilenum=itile)
-                src = os.path.join(config.DATA, 'anl', filename_in)
-                dest = os.path.join(config.DATA, 'anl', filename_out)
+                src = os.path.join(self.task_config.DATA, 'anl', filename_in)
+                dest = os.path.join(self.task_config.DATA, 'anl', filename_out)
                 inclist.append([src, dest])
             FileHandler({'copy': inclist}).sync()
 
         # loop over times to apply increments
         for bkgtime in bkgtimes:
             logger.info("Processing analysis valid: {bkgtime}")
             logger.info("Create namelist for APPLY_INCR_EXE")
-            nml_template = config.APPLY_INCR_NML_TMPL
+            nml_template = self.task_config.APPLY_INCR_NML_TMPL
             nml_config = {
                 'current_cycle': bkgtime,
-                'CASE': config.CASE,
-                'DATA': config.DATA,
-                'HOMEgfs': config.HOMEgfs,
-                'OCNRES': config.OCNRES,
+                'CASE': self.task_config.CASE,
+                'DATA': self.task_config.DATA,
+                'HOMEgfs': self.task_config.HOMEgfs,
+                'OCNRES': self.task_config.OCNRES,
             }
             nml_data = Jinja(nml_template, nml_config).render
             logger.debug(f"apply_incr_nml:\n{nml_data}")
 
-            nml_file = os.path.join(config.DATA, "apply_incr_nml")
+            nml_file = os.path.join(self.task_config.DATA, "apply_incr_nml")
             with open(nml_file, "w") as fho:
                 fho.write(nml_data)
 
             logger.info("Link APPLY_INCR_EXE into DATA/")
-            exe_src = config.APPLY_INCR_EXE
-            exe_dest = os.path.join(config.DATA, os.path.basename(exe_src))
+            exe_src = self.task_config.APPLY_INCR_EXE
+            exe_dest = os.path.join(self.task_config.DATA, os.path.basename(exe_src))
             if os.path.exists(exe_dest):
                 rm_p(exe_dest)
-            os.symlink(exe_src, exe_dest)
+            cp(exe_src, exe_dest)
```

Comment:
```suggestion
            os.symlink(exe_src, exe_dest)
```
Revert the change back, because the simple copy cannot find the support lib. The job failed as below:
`apply_incr.exe: error while loading shared libraries: libjediincr.so: cannot open shared object file: No such file or directory`